src/app/routes/applications/organization-list/organization-list.component.html

```diff
@@ -0,0 +1,114 @@
+<!-- Filter Sub Organization -->
```

Comment:
Does it make sense to you that this new `organization-list.component` is under `routes/applications`. What does "applications" mean in this context 🤔 ?

Reply:
No, it doesn't. I forgot to change application.component.ts into governance.component.ts. I should change also routing here. Thanks for pointing this out!